scripts/add-cluster.sh

```diff
@@ -75,6 +75,11 @@ rules:
   - "*"
 EOF
 else
+    CLUSTER_ROLE_NAME=${SA_NAME}-${OPERATOR_NS}-toolchaincluster
+    # we need to delete the binding since we cannot change the roleRef of the existing binding
+    if [[ -n `oc get ClusterRoleBinding ${CLUSTER_ROLE_NAME} 2>/dev/null` ]]; then
```

Comment:
the additional parameters should have been used also in the other case that you copy-pasted it from
```suggestion
    if [[ -n `oc get ClusterRoleBinding ${CLUSTER_ROLE_NAME} ${OC_ADDITIONAL_PARAMS} 2>/dev/null` ]]; then
```